kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/asset/WireAssetOCD.java

```diff
@@ -39,6 +41,13 @@ public class WireAssetOCD extends BaseAssetOCD {
     private static final String EMIT_ERRORS_DESCRIPTION = "Specifies wheter errors should be included or not "
             + "in the emitted envelope";
 
+    private static final String EMIT_ON_CHANGE_DESCRIPTION = "If set to true, this component will emit a value on Kura Wires"
+            + " only if it is different than the one obtained from the previous read operation or event for the corresponding channel."
```

Comment:
Is it better like this? "If set to true, this component will include a channel value in the output emitted in Kura Wires only if it value is different than the one from the previous read operation or event."